### ISSUE TITLE
InitializerPlugin type assertion failure

### ISSUE THREAD

**Issue by zhouzhuojie** [ovh/utask#149]
**Describe the bug**
Failed to assert type of plugin 'myinit.so': expected InitializerPlugin got *plugins.InitializerPlugin

**To Reproduce**

```sh
wget https://github.com/ovh/utask/releases/latest/download/install-utask.sh
sh install-utask.sh
mkdir init/myplugin
```

```go
// init/myplugin/main.go

package main                                                                  
                                                                              
import "github.com/ovh/utask/pkg/plugins"                                     
                                                                              
var Plugin = NewMyPlugin()                                                    
                                                                              
type MyPlugin struct{}                                                        
                                                                              
func NewMyPlugin() plugins.InitializerPlugin            { return &MyPlugin{} }
func (p *MyPlugin) Init(service *plugins.Service) error { return nil }        
func (p *MyPlugin) Description() string                 { return "myplugin" } 
```

```sh
docker-compose build
docker-compose up
```

```
Starting utask-test_db_1 ... done
Recreating utask-test_utask_1 ... done
Attaching to utask-test_db_1, utask-test_utask_1
db_1     | LOG:  database system was shut down at 2020-05-23 15:57:11 UTC
db_1     | LOG:  MultiXact member wraparound protections are now enabled
utask_1  | wait-for-it.sh: waiting 15 seconds for db:5432
utask_1  | wait-for-it.sh: db:5432 is available after 0 seconds
utask_1  | time="2020-05-23T15:59:18Z" level=fatal msg="failed to assert type of plugin 'myinit.so': expected InitializerPlugin got *plugins.InitializerPlugin"
db_1     | LOG:  database system is ready to accept connections
db_1     | LOG:  autovacuum launcher started
db_1     | LOG:  incomplete startup packet
utask-test_utask_1 exited with code 1
```

**Expected behavior**
It should pass the interface type assertion.

**utask version impacted by the bug**
v1.6.0

**Context which you are currently running utask**
Fresh installation from `install-utask.sh`

**Additional context**
Got the same error on Mac and Ubuntu.

**Comment by loopfz** [comment]
Thanks for the detailed report, will look into it and report back soon

**Comment by rbeuque74** [comment]
Hello @zhouzhuojie,
This issue should be resolved.
I released v1.6.1, currently building, you should not have problem with that.

Thanks again for the issue report (very clear and detailled :) ) !

**Comment by rbeuque74** [comment]
Regarding the last commits, I will release under 1.7.0, instead of 1.6.1
